spec/Centipede/Filter/UrlFilterSpec.php

```diff
@@ -29,6 +24,6 @@ function it_filters_url_hash()
 
     function it_filters_trailing_slash()
     {
-        $this->filter('https://github.com/')->shouldReturn('https://github.com');
+        $this->filter('https://github.com/')->shouldReturn('https://github.com/');
```

Comment:
`it_des_not_filter_out_trailing_slash` seems great, I'm gonna change it for this ;)